packages/cardano-graphql-services/src/TxSubmit/cli.ts

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+/* eslint-disable import/imports-first */
+require('../../scripts/patchRequire');
+import { Command } from 'commander';
+import { InvalidLoggerLevel } from '../errors';
+import { LogLevel, createLogger } from 'bunyan';
```

Comment:
Would be great to settle on a single logging library across entire SDK

Reply:
Not sure I understand. Do you mean to define a function in the utils package, so we control the external dependency, or is there another library I'm not familiar with? `ts-log` is just the abstract logger interface if that's what you mean.

Reply:
Never mind, I misunderstood the purpose of it